test/unit/materialized_view_fuel_test.sql

```diff
@@ -254,12 +273,174 @@ select results_eq(
+    ARRAY['non-biomass'::varchar],
+    'column fuel_classification in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select fuel_description from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY[''::varchar],
+    'column fuel_description in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select fuel_units from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['Sm^3'::varchar],
+    'column fuel_units in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select annual_fuel_amount from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['4550520'::varchar],
+    'column annual_fuel_amount in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select annual_weighted_avg_hhv from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['0.0369'::varchar],
+    'column annual_weighted_avg_hhv in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select annual_weighted_avg_carbon_content from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['0.7192'::varchar],
+    'column annual_weighted_avg_carbon_content in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select annual_steam_generation from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['.6'::varchar],
+    'column annual_steam_generation in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select alternative_methodology_description from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['wordswordswords'::varchar],
+    'column alternative_methodology_description in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select other_flare_details from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['flare!'::varchar],
+    'column other_flare_details in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select q1 from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY[1::varchar],
+    'column q1 in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select q2 from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['2'::varchar],
+    'column q2 in ggircs_swrs.fuel was properly parsed from xml'
+);
 
+select results_eq(
+    'select q3 from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['3'::varchar],
+    'column q3 in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select q4 from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['4'::varchar],
+    'column q4 in ggircs_swrs.fuel was properly parsed from xml'
+);
+
+select results_eq(
+    'select wastewater_processing_factors::text from ggircs_swrs.fuel where fuel_idx=0 and unit_idx=0',
+    ARRAY['<WastewaterProcessingFactors/>'::text],
```

Comment:
this would be a good candidate for an integration test: does one exist that is not self-terminating?